src/pages/events/[id]/repair-requests.tsx

```diff
@@ -0,0 +1,122 @@
+import { useEffect, useState } from "react";
+import { Inter } from "next/font/google";
+import { useRouter } from "next/router";
+import { RepairRequest } from "@prisma/client";
+import { CiCirclePlus } from "react-icons/ci";
+
+import Card from "@/components/Cards/card";
+import RepairAttemptForm from "@/components/Forms/repair-request-form";
+import { HeaderProps } from "@/components/Header";
+import Header from "@/components/Header";
+import Modal from "@/components/Modal";
+import Sidebar from "@/components/sidebar/index";
+import SearchBar from "@/components/ui/SearchBar";
+import SortBy from "@/components/ui/SortBy";
+const inter = Inter({ subsets: ["latin"] });
+
+export default function RepairRequest() {
```

Comment:
Eslint is throwing a warning here because the component name you're exporting is the same as the `RepairRequest` type from prisma. Consider renaming it